src/refactor_nrepl/artifacts.clj

```diff
@@ -17,6 +17,14 @@
 (defonce artifacts (atom {} :meta {:last-modified nil}))
 (def millis-per-day (* 24 60 60 1000))
 
+(defn- get-proxy-opts
+  "Generates proxy options from JVM properties for httpkit-client "
+  []
+  (if-let [proxy-host (some #(System/getProperty %) ["http.proxyHost" "https.proxyHost"])]
+    {:proxy-host proxy-host
+     :proxy-port (some->> ["http.proxyPort" "https.proxyPort"]
```

Comment:
the code prefers `http` proxy over `https`. perhaps the other way around would be a better idea?